File: examples/modulith-clinical-tool-jpa/src/main/java/io/zenwave360/example/clinicaltool/modules/masterdata/core/inbound/dtos/MasterDataKeyValue.java
package io.zenwave360.example.clinicaltool.modules.masterdata.core.inbound.dtos;

import io.zenwave360.example.clinicaltool.modules.masterdata.core.domain.*;
import java.io.Serializable;
import java.math.*;
import java.time.*;
import java.util.*;
import jakarta.validation.constraints.*;

/**
 * MasterDataKeyValue.
 */
@lombok.Getter @lombok.Setter
public  class MasterDataKeyValue  implements Serializable {


    
    private String key ;

    
    private String value ;





}

File: examples/modulith-klinical-tool-jpa/src/main/kotlin/io/zenwave360/example/clinicaltool/modules/masterdata/dtos/MasterDataKeyValue.kt
package io.zenwave360.example.clinicaltool.modules.masterdata.dtos

import io.zenwave360.example.clinicaltool.modules.masterdata.domain.*
import jakarta.validation.constraints.*
import java.io.Serializable
import java.math.*
import java.time.*

/** MasterDataKeyValue. */
data class MasterDataKeyValue(val key: String? = null, val value: String? = null) : Serializable {}
